### DISCUSSION TITLE
I am unable to view blockages after appyling

### DISCUSSION THREAD

**Discussion by Muthuluru-Sneha** [The-OpenROAD-Project/OpenROAD#6430] [Q&A]
Actually I want to apply blockage after routing due to more congestion at area {135 46 145 56 } so i used these commands
set block  [ord::get_db_block]
_40b5ff2de55b0000_p_odb__dbBlock
>>> set rect [odb::new_Rect 135 46 145 56]
_20c0d334e55b0000_p_odb__Rect
>>> set b [odb::dbBlockage_create $block \
...                  [$rect xMin] [$rect yMin] [$rect xMax] [$rect yMax]]
_4c283c31e55b0000_p_odb__dbBlockage
>>> set c [odb::dbBlockage_setSoft $b]
after i am not able to see blockage in gui .What to do below is the screenshot after applying blockage 
![Screenshot from 2024-12-25 13-19-53](https://github.com/user-attachments/assets/490f4bc4-14cd-4c7e-a2a4-83002f92d6b3)

**Comment by gadfort** [comment]
the rect and blockage take DBUs, not microns, as their inputs, so I suspect you have a tiny blockage in the lower left corner.
You need to use
`set rect [odb::new_Rect [ord::microns_to_dbu 135] [ord::microns_to_dbu 46] [ord::microns_to_dbu 145] [ord::microns_to_dbu 56]]`
